search/functions.py

```diff
@@ -8,3 +8,71 @@ def delete_document(index, doc_type, id):
     """
     es = connections.get_connection()
     es.delete(index=index, doc_type=doc_type, id=id, refresh=True)
+
```

Comment:
Not really relevant to this PR, just an aside, but to me, `functions` is too general a module name to be very useful. It seems kind of like `helpers` or `utils`? `delete_document` is pretty core though, so I'm not sure what an ideal name would be.

Reply:
I agree, let's call them helpers like in the `dips` app.